charts/add-ons/tracing/values.yaml

```diff
@@ -1,6 +1,28 @@
 # Default values for tracing.
 collector:
   image: omnition/opencensus-collector:0.1.11
-  # resources:
+  resources:
```

Comment:
I think the indentation is wrong here.  resources should have two children: cpu and memory.  each of those should have two children: limit and request.

Reply:
Well caught. Tradeoffs when running copy & paste quick and dirty 🥇